services/minespace-web/src/components/pages/Project/ProjectPage.js

```diff
@@ -227,13 +242,18 @@ export class ProjectPage extends Component {
           <Row gutter={[0, 16]}>
             <Col span={24}>
               <Tabs
-                defaultActiveKey={tabs[0]}
+                defaultActiveKey={this.props.match.params?.tab || tabs[0]}
                 onChange={(activeTab) => this.handleTabChange(activeTab, irtStatus)}
                 type="card"
               >
                 <Tabs.TabPane tab="Overview" key="overview">
                   <ProjectOverviewTab navigateForward={this.navigateFromProjectStagesTable} />
                 </Tabs.TabPane>
+                {this.props.isFeatureEnabled(Feature.AMS_AGENT) && (
+                  <Tabs.TabPane tab="Project Description" key="project-description">
+                    <ProjectDescriptionTab />
```

Comment:
Sorry, only noticed this now that I'm working in the same area. 😅 
BUT- this component, and the CORE equivalent, are the parent of these Tabs, right? And this parent always has the project loaded if `isLoaded`. 
I think you can just pass the project as a prop then and take the fetch out from the child component. One less fetch!

Reply:
I think these parent files only fetch on `componentDidMount` and I believe we should still need the project fetch inside ProjectDescriptionTab when a retry is attempted so we can also dispatch the changes.